db/hints/manager.cc

```diff
@@ -389,6 +389,14 @@ hint_endpoint_manager& manager::get_ep_manager(const endpoint_id& host_id, const
     }
 }
 
+uint64_t manager::max_size_of_hints_in_progress() const noexcept {
+    if (utils::get_local_injector().enter("decrease_max_size_of_hints_in_progress")) [[unlikely]] {
+        return 1'000;
+    } else {
+        return MAX_SIZE_OF_HINTS_IN_PROGRESS;
+    }
+}
```

Comment:
Can we put this function into the header? In release mode error injections are a no-op, so the optimizer should be able to inline this function. It's on a hot path (it's called in the write coordinator logic), so I think this matters.

Reply:
The constant is only used in `manager.cc`, so the function should be inlined

Reply:
> The constant is only used in `manager.cc`, so the function should be inlined. But I can move it to the header if that's preferable

No, if it's only used in manager.cc then it's fine. I thought that it might also be used in `storage_proxy.cc`.